WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -205,13 +205,8 @@ - (void)traitCollectionDidChange:(UITraitCollection *)previousTraitCollection
 {
     [super traitCollectionDidChange:previousTraitCollection];
 
-    if (@available(iOS 13.0, *)) {
-        // Update cached attributed strings when toggling light/dark mode.
```

Comment:
It might be nice to preserve this comment

Reply:
Added in 2cc9eeeb5b22f2b00149c02055dfa5b2ca8180ad